styles.css

```diff
@@ -0,0 +1,270 @@
+/* Задаем основные параметры страници */
+.body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  background: #E5E5E5;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E5E5E5" to be "#e5e5e5" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)